infra/charts/turing/templates/_helpers.tpl

```diff
@@ -80,6 +80,43 @@
 {{- .Values.global.sentry.dsn | default .Values.sentry.dsn -}}
 {{- end -}}
 
+{{- define "turing.plugins.directory" -}}
+/app/plugins
+{{- end -}}
+
+{{- define "turing.plugins.initContainers" -}}
+{{ if .Values.turing.experimentEngines }}
+initContainers:
+{{ range $expEngine := .Values.turing.experimentEngines }}
```

Comment:
this is to support multiple experiment engines at run time?

Reply:
That's nothing new, Turing can be deployed with multiple experiment engines, and it's up to users to decide which experiment engine to use during the router configuration step